### ISSUE TITLE
flow error when importing from react-navigation

### ISSUE THREAD

**Issue by arnoldbird** [react-navigation/react-navigation#3640]
### Current Behavior

I'm using flow and react-navigation in my React Native project.  I import components from react-navigation...

`import { TabNavigator, StackNavigator } from 'react-navigation'`

But with flow, in all files where I import from react-navigation, I'm getting a red underline beneath 'react-navigation' and this error...

```
[flow] Cannot resolve module `react-navigation`.
module "/Users/me/Library/Caches/typescript/2.7/node_modules/@types/react-navigation/index"
```

### Expected Behavior

I expect no flow error.  With the rest of my imports, there is no flow error.

### How to reproduce

Set up [flow](https://flow.org/en/) and then import components from react-navigation in one of your js files.

### Your Environment

In my package.json I have...

`"react-navigation": "git+https://github.com/react-community/react-navigation.git"`

This issue came up last spring, and people were able to fix it by requiring react-navigation from git, but in this case that's not fixing it.

Environment:
  OS: macOS High Sierra 10.13.3
  Node: 9.5.0
  Yarn: 1.3.2
  npm: 5.6.0
  Watchman: 4.9.0
  Xcode: Xcode 9.2 Build version 9C40b
  Android Studio: Not Found

Packages: (wanted => installed)
  react: 16.2.0 => 16.2.0
  react-native: https://github.com/expo/react-native/archive/sdk-25.0.0.tar.gz => 0.52.0

**Comment by Ashoat** [comment]
Run `flow-typed install`

**Comment by arnoldbird** [comment]
That did not resolve the flow error.

**Comment by Ashoat** [comment]
Something is wrong with your set-up then...

I am not sure what `composer.json`, but I assume you have some sort of custom set-up and that is your equivalent of a `package.json`? Can you try using a standard `react-navigation` release and see if that helps? If that doesn't help, can you let me know what `flow-typed/npm/react-navigation_v1.x.x.js` contains? Can you try running `flow-typed install react-navigation@1.0.0`? If none of that works, then please copy-paste your `.flowconfig` here for me.

**Comment by arnoldbird** [comment]
Today I'm not seeing the flow error.  Yesterday, I tried to clear the error by doing things like deleting my node_modules, running install again, closing out Visual Code Studio and re-opening, etc.  And the error didn't go away.  But today, I booted up and looked at my project and I see no more flow error in Visual Code Studio.  I'm not sure how this got resolved.

(My issue referred to "composer.json" but I meant to type package.json.  Sorry for any confusion.  I edited that part of the issue description just now.)
